.gitignore

```diff
@@ -0,0 +1,12 @@
+.idea
```

Comment:
Follow these steps:
1. git checkout main
2. git pull
3. git checkout t2
4. git merge main
5. git push

Reply:
This will put the latest code from `main` branch in the `t2` branch

Reply:
when you do these steps, look at this PR. you will notice lots of these files would have gone missing